### DISCUSSION TITLE
Customize Splash the Splash Image

### DISCUSSION THREAD

**Discussion by doctomlug** [freeplane/freeplane#1438] [Help] [answered]
Hello there,
I am running FreePlane on several machines and all of them are set up differently. To ensure I started the right version I like to modify the splash screen image but fail to find it in the program directory (Java noob!). 
- Where is it stored (which jar file)?
- I assume extracting, edit & updating the jar is enough to change it, correct?

Thanks for helping out.

**Comment by euu2021** [comment, marked as answer]
Another idea is to set a different Workspace title for each version: `Preferences…->Appearance->Look and feel->Workspace title`

**Reply by doctomlug** [reply]
It is amazing how much stuff is (well) hidden in the menus. Still new things to discover after such a long time.
Thanks for the good hint, this works perfectly.
